package.json

```diff
@@ -48,7 +48,7 @@
     "request": "2.88.0",
     "semver": "6.0.0",
     "verdaccio-audit": "1.2.0",
-    "verdaccio-htpasswd": "2.0.0-beta.1"
+    "verdaccio-htpasswd": "2.0.0"
```

Comment:
Let's keep this one in a different PR. I need this one now for beta release, so, please remove this commit to avoid conflicts.